library/Reporting/Web/Forms/ReportForm.php

```diff
@@ -34,27 +34,34 @@ protected function assemble()
         $this->setDefaultElementDecorator(new CompatDecorator());
 
         $this->addElement('text', 'name', [
-            'required' => true,
-            'label'    => 'Name'
+            'description' => $this->translate('A unique name of your report. '
+                . 'It will be used when exported to pdf, json or csv format and also when you list the reports in cli'),
```

Comment:
I would rephrase this as follows: 
`A unique name of this report. It is used when exporting to pdf, json or csv format and also when listing the reports in the cli`